assemblyline/templates/_helpers.tpl

```diff
@@ -353,4 +390,43 @@ data:
 {{ end }}
 {{ end }}
 
+{{ if .Values.configuration.retrohunt.enabled }}
+  {{with merge (dict "host" "hauntedhouse") . }}
+    {{template "assemblyline.GenCert" .}}
+  {{end}}
+
+  {{with merge (dict "host" "hauntedhouse-worker") . }}
+    {{template "assemblyline.GenCert" .}}
+  {{end}}
+{{ end }}
+
+
+# Certificate for service updaters
```

Comment:
Pretty sure this is old code we don't need anymore since Scaler is generating the certs for service dependencies